protractor.config.js

```diff
@@ -4,7 +4,7 @@ exports.config = {
     seleniumAddress: 'http://localhost:4444/wd/hub',
     specs: ['src/tests/**/*.spec.js'],
     jasmineNodeOpts: {
-        defaultTimeoutInterval: 60000
+        defaultTimeoutInterval: 30000
```

Comment:
We purposely increased the timeout to support long set of test. We were having random test failure otherwise.
@naina-verma please comment.